dev/devicelab/lib/tasks/microbenchmarks.dart

```diff
@@ -142,6 +142,12 @@ Future<Map<String, double>> _readJsonResults(Process process) {
       jsonStarted = false;
       processWasKilledIntentionally = true;
       resultsHaveBeenParsed = true;
+      // Sending a SIGINT/SIGTERM to the process here isn't reliable because [process] is
+      // the shell (flutter is a shell script) and doesn't pass the signal on.
+      // Sending a `q` is an instruction to quit using the console runner.
+      // See https://github.com/flutter/flutter/issues/19208
+      process.stdin.write('q');
+      await process.stdin.flush();
       // ignore: deprecated_member_use
       process.kill(ProcessSignal.SIGINT); // flutter run doesn't quit automatically
```

Comment:
I'm not sure which bit you're referring to - do you mean the comment on the .kill()?

We could remove the kill, but I think ultimately we want to fix the issue (it may require a VM change to allow snapshot validation so we can change the shell script to use `exec`) and go back to the `kill`. I can remove it for now and add some comments if you think that might make more sense though.

Reply:
I'm all for leaving the kill, just changing the comment to "in case flutter run doesn't quit automatically" or some such. The comment is wrong, now that we _do_ quit it.

Reply:
Ah sorry, I see what you mean now. The original comment was referring to the behaviour of `flutter run` not quitting automatically, but I agree it was a bit misleading without context. I've updated now (and swapped the comment/ignore around to make it more readable).